sdcm/cluster_k8s/mini_k8s.py

```diff
@@ -27,7 +27,7 @@
 from sdcm.cluster_k8s.iptables import IptablesPodPortsRedirectMixin, IptablesClusterOpsMixin
 from sdcm.cluster_gce import MonitorSetGCE
 from sdcm.utils.k8s import TokenUpdateThread, HelmValues
-from sdcm.utils.common import get_free_port, wait_for_port
+from sdcm.utils.common import get_free_port, wait_for_port, find_free_port
```

Comment:
I looked at the "get_free_port" func and looks like it does exactly what you need.
Can you please reuse it here?

Reply:
done